src/main/java/me/svdev/mscategory/dao/entity/Category.java

```diff
@@ -0,0 +1,53 @@
+package me.svdev.mscategory.dao.entity;
+
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "categories")
+public class Category {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(nullable = false, unique = true)
+    private Long id;
+
+    @Column(nullable = false, unique = true, length = 32)
+    private String name;
+
+    @Column(nullable = false)
+    private Boolean isVisible = true;
+
+    @Column(nullable = false)
+    private Boolean isFavorite;
```

Comment:
If you write isVisible = true; you need to write also isFavorite=false, but i recommended use default values in liquibase and add this settings to mapper